HTML/about.html

```diff
@@ -27,7 +27,7 @@
     <h1 class="text">About Me</h1>
     <div class="first-about">
         <div class="image">
-            <img src="../image/Teresa-300x271.jpg" width=""  alt="Teresa Raiford smilar photo">
+            <img src= "../image/Teresa .JPG"   alt="Teresa Raiford smilar photo">
```

Comment:
Is there a space before .jpg in the filename?

Reply:
I fix the space in the filename must of prest the space bar.